x-pack/test/api_integration/apis/cloud_security_posture/status/status_unprivileged.ts

```diff
@@ -247,8 +247,8 @@ export default function (providerContext: FtrProviderContext) {
           `expected unprivileged but got ${res.cspm.status} instead`
         );
         expect(res.vuln_mgmt.status).to.eql(
-          'not-installed',
-          `expected not-installed but got ${res.vuln_mgmt.status} instead`
+          'unprivileged',
```

Comment:
@animehart I wonder what led to this change in the test, because I don't see the change in the code with the intent to change that. Do you have an understanding of that?